examples/playground.js

```diff
@@ -23,12 +23,7 @@ const resolvers = {
 app.register(mercurius, {
   schema,
   resolvers,
-  graphiql: 'playground',
-  playgroundHeaders (window) {
-    return {
-      authorization: `bearer ${window.sessionStorage.getItem('token')}`
-    }
-  }
+  graphiql: true
```

Comment:
That's a good suggestion! We could pass serialized props object when accessing the `/graphiql/config.js` route but this would limit our ability to assigning the function props.

Is there a better approach I'm not thinking of?

Reply:
You could add a graphiqlProps / graphiqlOptions attribute of type GraphiQLProps in MercuriusCommonOptions that is serialized and passed inside a window.GRAPHIQL_PROPS prop when accessing the /graphiql/config.js route but